From 01b5478638c705af29f09ad4ed70289f70a21dcf Mon Sep 17 00:00:00 2001
From: "Yann E. MORIN" <yann.morin.1998@free.fr>
Date: Mon, 6 Feb 2017 14:04:47 +0100
Subject: [PATCH] p8-platform-config: fix lookup paths

Use the proper methods to find the headers and the library.

Signed-off-by: Samuel Martin <s.martin49@gmail.com>
Signed-off-by: "Yann E. MORIN" <yann.morin.1998@free.fr>
---
 p8-platform-config.cmake.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/p8-platform-config.cmake.in b/p8-platform-config.cmake.in
index 47f30f1..f59418d 100644
--- a/p8-platform-config.cmake.in
+++ b/p8-platform-config.cmake.in
@@ -10,7 +10,7 @@
 #
 # propagate these properties from one build system to the other
 set (p8-platform_VERSION "@p8-platform_VERSION_MAJOR@.@p8-platform_VERSION_MINOR@")
-set (p8-platform_INCLUDE_DIRS @p8-platform_INCLUDE_DIRS@ @CMAKE_INSTALL_PREFIX@/include)
+find_path (p8-platform_INCLUDE_DIRS p8-platform/os.h)
 set (p8-platform_LIBRARY_DIRS "@CMAKE_LIBRARY_OUTPUT_DIRECTORY@")
 set (p8-platform_LINKER_FLAGS "@p8-platform_LINKER_FLAGS@")
 set (p8-platform_CONFIG_VARS "@p8-platform_CONFIG_VARS@")
@@ -19,7 +19,7 @@ set (p8-platform_CONFIG_VARS "@p8-platform_CONFIG_VARS@")
 if(WIN32)
   set (p8-platform_LIBRARY "@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@/p8-platform.lib")
 else(WIN32)
-  set (p8-platform_LIBRARY "-L@CMAKE_INSTALL_PREFIX@/@CMAKE_INSTALL_LIBDIR@ -lp8-platform")
+  find_library (p8-platform_LIBRARY p8-platform)
 endif(WIN32)
 set (p8-platform_LIBRARIES ${p8-platform_LIBRARY} "@p8-platform_LIBRARIES@")
 mark_as_advanced (p8-platform_LIBRARY)
-- 
2.7.4

